test/storage/file.js

```diff
@@ -528,6 +528,44 @@ describe('File', function() {
       writable.write('data');
     });
 
+    it('should re-emit errors', function(done) {
+      var error = new Error('Error.');
```

Comment:
Love that my boring world's most boring error is catching on.

We should just have a custom Type, BoringTestError.